### PR TITLE
Guidance for external contributors

### PR DESCRIPTION
**Pull request by unzvfu**
Update README to provide guidance for external contributors with a particular view to managing the burden on PR reviewers.

### CODE REVIEW

**Comment by pgebheim** [comment]
For normal OSS project templates this info goes into CONTRIBUTING.md (and may be duplicated in the readme).

Also I think this warrants a PR template:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

File: README.md [hunk 1]
```diff
@@ -61,6 +61,112 @@ Licensed under either of
...
+really required to avoid duplication of work with other contributors.
+
+What follows are some more specific requests for how to write PRs in a
+way that will make them easy for us to review. Deviating from these
+guidelines may result in your PR being rejected, ignored or forgotten.
+
+
+### General guidance for your PR
+
+Obviously PRs will not be considered unless they pass the our Github
```
Comment (Nashtare) [inline README.md:102]:
```suggestion
Obviously PRs will not be considered unless they pass our Github
```

File: README.md [hunk 2]
```diff
@@ -61,6 +61,112 @@ Licensed under either of
...
+2. what turned out to the cause the bug; and
+3. how the PR fixes the bug.
+
+Wherever possible, PRs that fix bugs should include additional tests
+that (i) trigger the original bug and (ii) pass after applying the PR.
+
+
+#### The PR implements a new feature
+
+If you plan contribute an implementation of a new feature, please
```
Comment (Nashtare) [inline README.md:141]:
```suggestion
If you plan to contribute an implementation of a new feature, please
```

File: README.md [hunk 3]
```diff
@@ -61,6 +61,112 @@ Licensed under either of
...
 
 
-### Contribution
+## Guidance for external contributors
+
+Do you feel keen and able to help with Plonky3? That's great! We
+encourage external contributions!
+
+We want to make it easy for you to contribute, but at the same time we
+must manage the burden of reviewing external contributions.  We are a
```
Comment (Nashtare) [inline README.md:70]:
```suggestion
must manage the burden of reviewing external contributions. We are a
```

File: README.md [hunk 4]
```diff
@@ -61,6 +61,112 @@ Licensed under either of
...
+small team, and the time we spend reviewing external contributions is
+time we are not developing ourselves.
+
+We also want to help you to avoid inadvertently duplicating work that
+is already underway, or building something that we that we will not
```
Comment (Nashtare) [inline README.md:75]:
```suggestion
is already underway, or building something that we will not
```

**Pull request merged**